CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
   - If you are setting global job listeners please also add `SentryJobListener`
 - Ensure serialVersionUID of Exception classes are unique ([#3115](https://github.com/getsentry/sentry-java/pull/3115))
 - Get rid of "is not eligible for getting processed by all BeanPostProcessors" warnings in Spring Boot ([#3108](https://github.com/getsentry/sentry-java/pull/3108))
+- Fix missing `release` and other fields for ANRs reported with `mechanism:AppExitInfo` ([#3074](https://github.com/getsentry/sentry-java/pull/3074))
```

Comment:
is this change wanted?

Reply:
Yeah, I accidentally fixed this issue, because we had a flaky failing test, so I think it's fine if it goes into this PR. Or do you think we need a separate PR for that?

Reply:
nah, it's fine, just wanted to be sure it wasn't a reminiscence of another WIP PR 👍